src/component/Beers.js

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+import { useState } from "react";
+import { useEffect } from "react";
+import { NavLink } from "react-router-dom";
+import Header from "./Header";
+
+function Beers() {
+
+const [beers, setBeers] = useState([]);
+
+useEffect(() => {
+    axios.get('https://ih-beers-api2.herokuapp.com/beers')
+    .then( response => {
+        setBeers(response.data)  
+    })
+    .catch(err => console.log('Error getting data from API', err))
+    }, [])
+
+ return (
+       
+        <div >
+            <Header/>
+            {beers.map(e => (
+                <div>
+                     <img src={e.image_url} alt='beer'/>
+                        <NavLink key={e._id} to={`/beers/${e._id}`}>
+                            <h1>{e.name}</h1>
+                                </NavLink>
+                            <p>{e.tagline}</p>
+                        <h6>Created by: {e.name}</h6>
+                     <hr/>
+                </div>
+            ))}
```

Comment:
With react  as you can see it takes a moment to receive info, so for the user it is nice to have a message or see 'spinner' while info is loading. Just a suggestion!

In this case you would use conditional rendering , for example:

```
      {beers.length === 0 
         ? <p>Data is loading</p>
         : { beers.map(e => (<div>rendering beers here etc</div>) }
      }
```